scheduler/src/cook/task.clj

```diff
@@ -47,6 +47,13 @@
           task-entity->compute-cluster-name
           cc/compute-cluster-name->ComputeCluster))
 
+(defn get-ComputeCluster-for-task-ent-if-present
```

Comment:
I would favor `get-compute-cluster-for-task-ent-if-present`.

Reply:
updated